app/src/main/java/com/rob729/quiethours/util/StoreSession.kt

```diff
@@ -30,6 +32,16 @@ object StoreSession {
         return sharedPreferences.getInt(key, 0)
     }
 
+    fun writeBoolean(key: String, value: Boolean) {
```

Comment:
This implementation will cause problems in the future as when a boolean value will be required to store in shared pref it will be stored under appSharedPrefs instead of sharedPreferences.
Revert the appSharedPrefs implementation.

Reply:
Or do one thing. Make it a more specific one. Instead of writeBoolean make it setNightMode and getNightMode.

Reply:
Ok

Reply:
@rob729 Done